install/kubernetes/cilium/values.yaml

```diff
@@ -675,6 +675,9 @@ hubble:
     #
     tolerations: []
 
+    # -- hubble-relay priorityClassName
```

Comment:
This ends up in the Documentation too, so maybe use something slightly more descriptive? Some similar fields use:
```
The priority class to use for ...
```

By the way, you must also build the documentation locally (`make -C Documentation`) and commit the changes from generated file Documentation/helm-values.rst. The CI will likely complain when it runs otherwise.

Reply:
@qmonnet thanks. The docs is updated